doc/whats_new.rst

```diff
@@ -44,6 +44,12 @@ Enhancements
 Bug fixes
 .........
 
+   - Fix a bug where :class:`sklearn.feature_selection.SelectFdr` did not 
+     exactly implement Benjamini-Hochberg procedure. It formerly may have
+     selected fewer features than it should.
+     (`#7490 <https://github.com/scikit-learn/scikit-learn/pull/7490>`_) by
+     `Peng Meng`_.
```

Comment:
You need to add yourself at the bottom of the page if you want your name to be a link.